core-java-modules/core-java-8-datetime-2/src/test/java/com/baeldung/parsingDates/SimpleParseDateTest.java

```diff
@@ -0,0 +1,40 @@
+package com.baeldung.parsingDates;
+
+import com.baeldung.parsingDates.SimpleDateTimeFormat;
+import com.baeldung.parsingDates.SimpleDateTimeFormater;
+import com.baeldung.parsingDates.SimpleDateUtils;
+import com.baeldung.parsingDates.SimpleParseDate;
+import java.util.Arrays;
+import org.junit.*;
+
+public class SimpleParseDateTest {

```

Comment:
Looks like this file name is causing the build to fail: https://jenkins.baeldung.com/job/tutorials-build-pr/15338/console

I think we need to rename it.

```suggestion
public class SimpleParseDateUnitTest {

```